### DISCUSSION TITLE
Turn on the relay every hour for 15 minutes

### DISCUSSION THREAD

**Discussion by skyvladik** [arendst/Tasmota#17144] [Q&A] [answered]
Hello. Help me write a rule. It is necessary to turn on the relay every hour for 15 minutes. Thanks.

**Comment by rapi3** [comment, marked as answer]
This will set relay to switch OFF after 15 minute.
pulsetime 1000

This rule will set relay ON every hour, don't forget to enable rule with rule1 1:
Rule1 ON Time#Minute|60 DO POWER 1 ENDON

**Reply by skyvladik** [reply]
Hello! Does Pulsetime 1000 work on all relays? Is it possible to use pulsetime1 1000, for example for the first relay. I want to use several relays.

**Reply by sfromis** [reply]
Yes - do read the docs for the [PulseTime](https://tasmota.github.io/docs/Commands/#pulsetime) command.

**Reply by rapi3** [reply]
Pulsetime1 till Pulsetime8 can & will be set independent for each 1-8 relay, please check the documentation: if you have more than 8 relays:
https://tasmota.github.io/docs/Commands/#pulsetime

**Comment by skyvladik** [comment]
Thanks! I figured out how to do it. But how to close the topic?

**Reply by sfromis** [reply]
You've already marked it as answered, close does not apply.

**Reply by skyvladik** [reply]
Thanks!
